net.casimirlab.frigoligo.yml

```diff
@@ -0,0 +1,40 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/main/data/flatpak-manifest.schema.json
+
+app-id: net.casimirlab.frigoligo
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: frigoligo
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+modules:
+  - name: Frigoligo
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      - mkdir -p /app/bin
+      - tar -xvf frigoligo-v1.0.10-linux-x64.tar.gz -C /app/
+      - chmod +x /app/frigoligo/frigoligo
+      - ln -s /app/frigoligo/frigoligo /app/bin/frigoligo
+      - mkdir -p /app/share/icons/hicolor/512x512/apps /app/share/applications /app/share/metainfo
+      - cp play_store_512.png /app/share/icons/hicolor/512x512/apps/net.casimirlab.frigoligo.png
+      - cp net.casimirlab.frigoligo.desktop /app/share/applications/
+      - cp net.casimirlab.frigoligo.metainfo.xml /app/share/metainfo/
+    sources:
+      - type: file
+        url: https://github.com/casimir/frigoligo/releases/download/v1.0.10/frigoligo-v1.0.10-linux-x64.tar.gz
```

Comment:
so how will that work on aarch64?

Reply:
It doesn't. I though the following lines prevent this case. https://github.com/flathub/flathub/blob/e75f33ea3014d273dc641138177ef07a3460d24a/net.casimirlab.frigoligo.yml#L19-L20

Reply:
I misunderstood the documentation then and that was not the intention. I can't find anything about architecture support, is it mandatory to support both `x86_64` and `aarch64`?

Reply:
This has been changed.